app/services/process_payment_intent.rb

```diff
@@ -36,10 +36,12 @@ def call!
     process_payment
 
     if payment.reload.completed?
-      payment.mark_as_processed
+      payment.completed_authorization
+      payment.clear_authorization_url
```

Comment:
I kept these as two separate methods for now; `clear_authorization_url` could potentially be an `after_transition` on `completed_authorization`.

Reply:
I think I would move it there. We want it to happen whenever that transition completes :+1: